apps/api/src/server/applications/application/project-updates/project-updates.controller.js

```diff
@@ -0,0 +1,35 @@
+import { DEFAULT_PAGE_NUMBER, DEFAULT_PAGE_SIZE } from '../../constants.js';
+import { listProjectUpdates } from '../../../repositories/project-update.respository.js';
+import { getPageCount } from '../../../utils/database-pagination.js';
+import { mapProjectUpdate } from './project-updates.mapper.js';
+import logger from '../../../utils/logger.js';
+
+/**
+ * @type {import('express').RequestHandler}
+ */
+export async function getProjectUpdates(req, res) {
+	const page = Number(req.query.page) || DEFAULT_PAGE_NUMBER;
+	const pageSize = Number(req.query.pageSize) || DEFAULT_PAGE_SIZE;
+	const caseId = parseInt(req.params.id);
+	logger.debug({ caseId, page, pageSize }, 'getProjectUpdates');
+	let orderBy;
+
+	if (req.query.sortBy) {
```

Comment:
Looks like there are multiple implementations of sortBy, representations.controller.js also has one, my be good to re-use the sorting and paging logic in the future.

Reply:
Consolidated into a util: https://github.com/Planning-Inspectorate/back-office/pull/992/commits/2fad469ef92a434ce459fe225020248d1593197e